Directory.Build.targets

```diff
@@ -38,7 +38,7 @@
         
         <PackageReference Update="Duende.IdentityServer" Version="$(IdentityServerVersion)" />
         
-        <PackageReference Update="CsQuery.NETStandard" Version="1.3.6.1" />
+        <PackageReference Update="AngleSharp" Version="1.1.2" />
```

Comment:
We're not using this yet, but I think that's fine.

Reply:
I started replacing CSQuery with AngleSharp then realised the CSQuery code wasn't being used at all. I don't want unused package refs so will remove this. Ty.